src/Tools/dotnet-monitor/CollectionRules/Options/Triggers/AspNetRequestCountOptionsDefaults.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options.Triggers
+{
+    internal static class AspNetRequestCountOptionsDefaults
+    {
+        public const string SlidingWindowDuration = "00:01:00";
```

Comment:
I noticed that we define this constant with the same value in four different places; would it make sense to have this as a shared default?

Reply:
I'll add another shared constant that these constants can reference. I'd like to maintain the aspects of each trigger and action separately as much as possible, but not at the expense of fixing an issue multiple times, as you pointed out.